osu.Game/Rulesets/Objects/HitObjectLifetimeEntry.cs

```diff
@@ -94,6 +94,6 @@ internal bool KeepAlive
         /// <summary>
         /// Set <see cref="LifetimeEntry.LifetimeStart"/> using <see cref="InitialLifetimeOffset"/>.
         /// </summary>
-        private void setInitialLifetime() => LifetimeStart = HitObject.StartTime - InitialLifetimeOffset;
+        public void SetInitialLifetime() => LifetimeStart = HitObject.StartTime - InitialLifetimeOffset;
```

Comment:
Could maybe be `internal`, unless there is a good reason for public exposition.

Reply:
I was about to say that I try to make sure anything we create can be replicated by a ruleset - so someone should be able to implement a SHOC themselves. But realised that `KeepAlive` is also internal, so... I've made the change.